src/Components/Moormanage/Boatyards/AddBoatyards.tsx

```diff
@@ -82,55 +163,76 @@ const AddBoatyards = () => {
           </div>
         </div>
 
-        <div>
-          <div className="bg-[#ECECEC] rounded-md border-[1px] p-1 border-gray-300 w-[40vw] mt-10">
-            <DataTable
-              header={'Existing Sites'}
-              tableStyle={{
-                fontSize: '12px',
-                color: '#000000',
-                fontWeight: 600,
-                backgroundColor: '#D1D1D1',
-              }}
-              size="small">
-              <Column header="" field="id" style={{ width: '4vw' }}></Column>
-              <Column style={{ width: '4vw' }} field="customerName" header=""></Column>
-              <Column style={{ width: '10vw' }} field="email" header=""></Column>
-              <Column style={{ width: '8vw' }} field="phone" header=""></Column>
-              <Column style={{ width: '8vw' }} field="address" header=""></Column>
-            </DataTable>
-
-            <Button
-              style={{
-                width: '100%',
-                height: '4vh',
-                backgroundColor: '#ECECEC',
-                cursor: 'pointer',
-                color: '#6B6B6B',
-                fontWeight: 'bolder',
-                fontSize: '0.50rem',
-                display: 'flex',
-                alignItems: 'center',
-                justifyContent: 'center',
-              }}
-              label="">
-              <img
-                src="/assets/images/plus.png"
-                alt="icon"
-                className="w-3 mr-1"
+        <div className="flex gap-8 mt-4">
+          <div>
+            <div className="mt-2">
+              <Dropdown
+                value={country}
+                onChange={(e) => setCountry(e.target.value)}
+                placeholder="Country"
                 style={{
-                  color: 'black',
-                  fontWeight: 'bolder',
+                  width: '13vw',
+                  height: '4vh',
+                  border: '1px solid gray',
+                  borderRadius: '0.50rem',
+                  fontSize: '0.80vw',
                 }}
               />
-              ADD NEW
-            </Button>
+            </div>
+          </div>
+
+          <div>
+            <div>
+              <div className="mt-2">
+                <InputComponent
+                  value={zipCode}
+                  onChange={(e) => setZipCode(e.target.value)}
+                  placeholder="Zip code"
+                  style={{
+                    width: '13vw',
```

Comment:
remove repetition of code by create compoenent

Reply:
already using CustomInputComponent here , code for styling is repeating here , i tried like this now -> 
  const style = {
    width: '13vw',
    height: '4vh',
    border: '1px solid gray',
    borderRadius: '0.50rem',
    fontSize: '0.80vw',
  }